code/modules/antagonists/gang/outfits.dm

```diff
@@ -0,0 +1,56 @@
+/datum/outfit/families_police/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+
+	var/obj/item/card/id/W = H.wear_id
+	W.access = get_all_accesses() // I have a warrant.
+	W.assignment = "Space Police"
```

Comment:
ah yes, you'll definitely be able to go undercover and infiltrate the sec department when your ID literally says "Space Police" on it.

they're undercover, don't change the name of their job, just give them all access

Reply:
if you check the undercover cop's datum, they aren't given the equipment pass

Reply:
and maybe give them an AA ID that looks like an assistant ID if they don't have their ID on, for whatever reason (maybe they were changing clothes when the gang announcement came?)